go.mod

```diff
@@ -12,8 +12,9 @@ require (
 	github.com/eapache/go-resiliency v0.0.0-20160104191539-b86b1ec0dd42
```

Comment:
Perhaps running the linting script updated your `go.mod` and `go.sum`? Please don't include those changes in the PR.

Reply:
Okay this is messed up